test/loco/integer/sendmoremoney2.clj

```diff
@@ -0,0 +1,54 @@
+(ns ^{:author "Andrea Richiardi"
+      :doc "Send + More = Money example solution #2 + tests. This solution takes the long road of
+            having a var for each letter, constraints on the addends of the sum plus carries and
+            does not make use of loco's $scalar constraint (see sendmoremoney.clj for a more concise
+            implementation)." }
+  loco.integer.sendmoremoney2
+  (:refer-clojure :exclude [send]) 
+  (:require [clojure.test :refer :all]
+            [loco.core :refer :all]
+            [loco.constraints :refer :all]))
+
+(defn initialize-digits [vars]
+  (for [v vars]
+    ($in v 0 9)))
+
+(def letters
+  [:S :E :N :D :M :O :R :Y])
+
+(def carries
+  (for [i (range 4)]
+    ($in [:_c i] 0 1)))
+
+(def ^{:doc "Model #2. Constraints on the addends with carries:
```

Comment:
`def` allows you to write docstrings for your vars. e.g.

    (def x
      "x is a var"
      1)